### PR TITLE
[TACHYON-255] Update documentation for Swift UFS setup.

### PR DESCRIPTION
**Pull request by calvinjia**
https://tachyon.atlassian.net/browse/TACHYON-255

### CODE REVIEW

File: docs/Setup-UFS.md [hunk 1]
```diff
@@ -126,48 +126,55 @@ and Hadoop 2.4.0 use.
 After the built is successful, there is need to configure `core-sites.xml` required by the Swift
 driver. The configuration template included in the `/conf/core-sites.xml.template` and it contains
 three example sections: local Swift based on Keystone authentication model, local Swift based on
-temp authentication model , SoftLayer public object store. The general structure of the parameters
+temp authentication model, and SoftLayer public object store. The general structure of the parameters
 are `fs.swift.service.<PROFILE>.<PARAMETER>` where `<PROFILE>` is any name that will be later used
-as a part of the  Swift URL. For example, if `<PROFILE>` is “swift1” than Swift URL would be
+as a part of the Swift URL. For example, if `<PROFILE>` is “swift1” than Swift URL would be
```
Comment (apc999) [inline docs/Setup-UFS.md]: s/than/then

File: docs/Setup-UFS.md [hunk 2]
```diff
@@ -126,48 +126,55 @@ and Hadoop 2.4.0 use.
...
 Where `testcont` is the existing Swift container and `swift1` is a profile in `core-sites.xml`.
-By default, Tachyon uses `/conf` directory to load `core-sites.xml`.  To specify different location,
+By default, Tachyon uses `/conf` directory to load `core-sites.xml`. To specify a different location,
 please configure `tachyon.underfs.hadoop.configuration` accordingly.
 
 ### Local file system usage
 
 By default, Tachyon uses local file system to store certain temporary files. In deployments where
-local file system is not exists, both  `tachyon.master.journal.folder` and
+local file system is not exists, both `tachyon.master.journal.folder` and
```
Comment (apc999) [inline docs/Setup-UFS.md]: does not exist

File: docs/Setup-UFS.md [hunk 3]
```diff
@@ -126,48 +126,55 @@ and Hadoop 2.4.0 use.
...
 `tachyon.master.temporary.folder` has to be configured with `swift://` name space.
 
 ### Accessing IBM SoftLayer object store
 
-To access SoftLayer there is need in additional preliminary step. Up to date, hadoop-openstack
+Using the Swift module also makes the IBM SoftLayer object store an option as an under storage system for
+Tachyon. To access SoftLayer there is need in additional preliminary step. Up to date, hadoop-openstack
```
Comment (apc999) [inline docs/Setup-UFS.md]: there is need ->  there needs ?

File: docs/Setup-UFS.md [hunk 4]
```diff
@@ -126,48 +126,55 @@ and Hadoop 2.4.0 use.
...
-By default, Tachyon uses `/conf` directory to load `core-sites.xml`.  To specify different location,
+By default, Tachyon uses `/conf` directory to load `core-sites.xml`. To specify a different location,
 please configure `tachyon.underfs.hadoop.configuration` accordingly.
 
 ### Local file system usage
 
 By default, Tachyon uses local file system to store certain temporary files. In deployments where
-local file system is not exists, both  `tachyon.master.journal.folder` and
+local file system is not exists, both `tachyon.master.journal.folder` and
 `tachyon.master.temporary.folder` has to be configured with `swift://` name space.
```
Comment (apc999) [inline docs/Setup-UFS.md]: has -> have?

**Comment by apc999** [comment]
There are a few typos in existing doc, could you fix them? Thanks for the update.

**Comment by calvinjia** [comment]
@apc999 thanks, I've addressed the comments.

**Comment by apc999** [comment]
LGTM +1

**Pull request merged**